### ISSUE TITLE
Narrator: Control type for 'More tab item' button is defined as menu item.

### ISSUE THREAD

**Issue by sanket856** [microsoft/vscode#137807]
[4975_More tab control type.zip](https://github.com/microsoft/vscode-docs/files/7594229/4975_More.tab.control.type.zip)
GitHubTags:#A11ySev3;#A11yMAS;#DesktopApp;#Visual Studio Code Client;#Win10;#screenreader;#Narrator;#WCAG4.2.1:#VisualStudioCode-Win32-May2021;#Linked:Bug28783700;

### Environment Details:​
Visual Studio Code.
Version: 1.62.3.

### Pre-Requisites:

1. 'Install 'Visual Studio Code' application.
2. Go to 'Extensions' and add 'Microsoft Edge(Chromium) Tools' extension.

### Repro Steps:​

1. Launch VS code application
2. Navigate to 'Microsoft Edge(Chromium) Tools' tab item and hit enter key.
3. Navigate to 'Open a New Tab' button and hit enter key.
5. Now navigate back to visual studio code application using ALT+TAB key or minimize the browser.
6. Navigate to 'More tab item' in style panel and listen to narrator's announcement.

### Actual Result:
Control type for 'More tab item' button is defined as menu item.

**NVDA Behavior:** This issue is not observed with NVDA as NVDA announces "More Tabs Menu button submenu".

### Expected Result:
Control type for 'More tab item' should be defined as button. And narrator should announce status as 'Expanded/Collapsed' upon invoking 'more tab item'.

### User Impact:
People who rely on screen reader will face difficulty to understand the functionality of 'More tab item' if control type is defined as menu item.

**Comment by isidorn** [comment]
Dup of https://github.com/microsoft/vscode/issues/10729
Narrator simply does not work well with Electron apps. This is a narrator issue which we can not fix on the VS Code side

**Comment by sbatten** [comment]
@sanket856 it is my understanding that this item can only be closed by the vendor team to reflect appropriately elsewhere. This item was originally created in the wrong repository and created an item our KPIs that we cannot remove. Can you help?
